extensions/amp-ad-network-doubleclick-impl/0.1/test/test-doubleclick-rtc.js

```diff
@@ -305,7 +305,6 @@ describes.realWin('DoubleClick Fast Fetch RTC', {amp: true}, env => {
         'json': JSON.stringify(json),
       });
       env.win.document.body.appendChild(element);
-      env.win.document.referrer = 'https://www.google.com/';
```

Comment:
That change was intentional. It turns out that the test is badly written, and fails when the referrer is set to `https://www.google.com`. Earlier, it was silently passing because the referrer was never being set.

Unless I'm missing something else?